viewing_party/party.py

```diff
@@ -1,23 +1,166 @@
-# ------------- WAVE 1 --------------------
+from pprint import pprint
+#HELPER FUNCTION
+def _get_movies_watched_by_friends(user_data):
+    movies_watched_by_friends = []
+    
+    for friend in user_data["friends"]: #[{}]

```

Comment:
Not sure what the purpose of the comment at the end of this line is, but it's safe to delete it.